src/screens/Components.js

```diff
@@ -365,4 +416,31 @@ const styles = StyleSheet.create({
     borderBottomRightRadius: theme.SIZES.BASE * 0.5,
     borderBottomLeftRadius: theme.SIZES.BASE * 0.5,
   },
+  slider1: {
```

Comment:
Too many JSS rules. All 9 JSS classes have the same rule `marginTop: 10`, try to put them in a single JSS class.